### PR TITLE
Teacher, AssignmentsTests

### PR DESCRIPTION
**Pull request by ndrsszsz**
[Bitrise - Canvas iOS - Teacher - E2ETests](https://app.bitrise.io/build/423567a2-939b-490f-aff5-09362cab0ac2)

refs: MBL-17050
affects: Teacher
release note: None
test plan: Tests to pass

### CODE REVIEW

**Comment by inst-danger** [comment]
<details><summary>Teacher Build QR Code:</summary>

</details>

**Comment by inst-danger** [comment]
#### Release Note: 
None

#### Affected Apps: Teacher

[MBL-17050](https://instructure.atlassian.net/browse/MBL-17050)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 92.46% | 92.46% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 06b642e970a24f706999d27785c95231ec9cd110
</p>

**Review by vargaat — APPROVED** [review]

File: rn/Teacher/ios/TeacherE2ETests/CourseDetails/CourseDetailsTests.swift
Comment (vargaat) [inline rn/Teacher/ios/TeacherE2ETests/CourseDetails/CourseDetailsTests.swift:1]: Is this a merge error?

**Review by vargaat — CHANGES_REQUESTED** [review]

File: rn/Teacher/ios/TeacherE2ETests/CourseDetails/CourseDetailsTests.swift
Comment (ndrsszsz) [reply rn/Teacher/ios/TeacherE2ETests/CourseDetails/CourseDetailsTests.swift:1]: Kind of.

**Review by vargaat — APPROVED** [review]

**Pull request merged**